x/qgb/keeper/keeper_data_commitment.go

```diff
@@ -85,22 +89,26 @@ func (k Keeper) GetDataCommitmentForHeight(ctx sdk.Context, height uint64) (type
 			return *dcc, nil
 		}
 	}
-	return types.DataCommitment{}, errors.Wrap(types.ErrDataCommitmentNotFound, "data commitment for height not found")
+	return types.DataCommitment{}, errors.Wrap(types.ErrDataCommitmentNotFound, "data commitment for height not found or was pruned")
 }
 
 // GetLastDataCommitment returns the last data commitment.
 func (k Keeper) GetLastDataCommitment(ctx sdk.Context) (types.DataCommitment, error) {
```

Comment:
[nit][not-blocking] proposal to conform to the earliest/latest naming pattern because last is a bit confusing (is it most recent or least recent?)

```suggestion
// GetLastDataCommitment returns the last data commitment.
func (k Keeper) GetLatestDataCommitment(ctx sdk.Context) (types.DataCommitment, error) {
```

Also this conforms to `GetLatestValset`

Reply:
https://github.com/celestiaorg/celestia-app/issues/1821